### ISSUE TITLE
Radio ID in YSF mode

### ISSUE THREAD

**Issue by Skinoku** [nostar/dudestar#72]
Hello and congratulations for the work done.
When transmitting in YSF mode on a reflector, if that reflector is linked to WIRES-X room, the traffic generated via this software will not go trough the Yaesu WIRES server because it's missing the RADIO ID.
Is it possible to add a filed in order to input a radio ID making the transmission go trough?
Thanks

**Comment by nostar** [comment]
I did the original changes to the DMR2YSF and friends to add the radio ID for DMR to YSF cross modes that had broken with Wires-X softwre updates at that time.  I later found that the radio ID is not needed at all to cross Wires-X.  The combination of a FICH CM (CallMode) value of 0 with the correct sequence of DT1 bytes is the key to crossing Wires-X.  I apply the same settings to Dudestar/Droidstar and there are no plans to make these settings editable at this time.  I have tested this on many YSF reflectors that are bridges to Wires-X nodes and there are no issues.  The details of these changes are discussed here:

https://forum.pistar.uk/viewtopic.php?f=24&t=2151
